deployment/copilot/kafka-broker/manifest.yml

```diff
@@ -0,0 +1,51 @@
+# The manifest for the "kafka-broker" service.
+# Read the full specification for the "Load Balanced Web Service" type at:
+#  https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/
+
+# Your service name will be used in naming your resources like log groups, ECS services, etc.
+name: kafka-broker
+#type: Backend Service
+type: Load Balanced Web Service
+
+## Distribute traffic to your service.
+http: false
+nlb:
+  port: 9092
+  
+# Allow service-to-service communication with ECS Connect
+network:
+  connect: true
+
+# Configuration for your containers and service.
+image:
+  location: docker.io/bitnami/kafka:3.4
+  # Port exposed through your container to route traffic to it.
+  port: 9092
+
+cpu: 256       # Number of CPU units for the task.
+memory: 2048    # Amount of memory in MiB used by the task.
+count: 1       # Number of tasks that should be running in your service.
+exec: true     # Enable running commands in your container.
```

Comment:
This comment is too vague - without it you'll still run at least ONE command in the container, right (entrypoint)